source/includes/k8s/deploy-operator.rst

```diff
@@ -74,22 +102,21 @@ The Kubernetes TLS API uses the CA signature algorithm for generating new TLS ce
 MinIO recommends ECDSA (e.g. `NIST P-256 curve <https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-4.pdf>`__) or EdDSA (e.g. :rfc:`Curve25519 <7748>`) TLS private keys/certificates due to their lower computation requirements compared to RSA.
 See :ref:`minio-TLS-supported-cipher-suites` for a complete list of supported TLS Cipher Suites.
 
-The Operator cannot complete initialization if the Kubernetes cluster is 
-not configured to respond to a generated CSR. Certain Kubernetes 
-providers do not specify these configuration values by default. 
+The Operator cannot complete initialization if the Kubernetes cluster is not configured to respond to a generated CSR. 
+Certain Kubernetes providers do not specify these configuration values by default. 
```

Comment:
Do we know which providers these `certain` ones are? As in, is there reasonable guidance we could offer?

Maybe change `certain` to `some` too.

Reply:
Too many K8s providers to try to keep up with whether they do or do not. We don't want to get into a naming game. But swapping for `some`, as suggested.